src/cli/commands/start.rs

```diff
@@ -44,12 +57,62 @@ impl Runnable for StartCmd {
     fn run(&self) {
         let config = APP.config();
 
-        abscissa_tokio::run(&APP, async {
-            match client::start(
+        abscissa_tokio::run_with_actix(&APP, async {
+            let sender = Account::new(config.faucet.mnemonic.clone(), &config.chain.prefix)
+                .expect("💀 Cannot create faucet account");
+
+            let addr_discord_client = DiscordActor::new(config.discord.token.to_string()).start();
+
+            let addr_cosmos_client = Client::new(APP.config().chain.grpc_address.to_string())
+                .await
+                .map_err(|err| {
+                    error!("💀 Cosmos GRPC client error: {:?}", err);
+                })
+                .unwrap()
+                .start();
+
+            let addr_tx_handler = TxHandler::<MsgSend>::new(
+                config.chain.chain_id.to_string(),
+                sender.clone(),
```

Comment:
The transaction handler shall be the only one carrying the account signing info.
```suggestion
                sender.toOwner(),
```